### DISCUSSION TITLE
[REQUEST]: Pathology Functions

### DISCUSSION THREAD

**Discussion by kathywake23** [PhenoApps/Field-Book#1024] [Ideas]
### Description

I'm collecting the same data over several time points. It would be nice to be able to bundle traits by time point (or by any variable) in the trait list. For example, say I need the traits "date", "time", and "color" for 10 samplings separated by time. I'm currently at time point/sampling 8. The current system has me going in and copying each trait individually every single time I need to start a new round of sampling. It would be great if Fieldbook allowed the user to group/bundle used traits out of the way and bulk copy bundled traits.

Grouping the traits would also allow for segregation between "fields". I take basically the same data over several time points on different plants in different environments. If I wanted to use the same device, the current Fieldbook system requires that I individually copy all of my traits and label them to be specific to a field. This can get very confusing, very fast. The only way I can think of at the moment to keep "fields" segregated from each other is to have Fieldbook on 2 separate devices. Having 2 devices (3 if you count my cellphone) when its 10O degF outside and 80% humidity is quite cumbersome.

**Comment by trife** [comment]
Similar to #566 and #66 so I'm going to convert this to a discussion. We'll be working on the field-based trait lists for v6.1. Time series-based data is going to take a little more time.
